workflows/cmd/generate-config-schema.ts

```diff
@@ -0,0 +1,7 @@
+import * as fs from "fs";
+import zodToJsonSchema from "zod-to-json-schema";
+import { Config } from "../src/config";
```

Comment:
Would it be easy to add (a version of) this file to the PR? So we can see what sort of things would be written to the schema?

Reply:
@tenwit I think I committed this file too early. It should have been part of the next PR, together with the schema config file you ask for, where I would go into reading a repository specific config file, with schema validation.